common/app/routes/Challenges/utils/index.js

```diff
@@ -24,7 +24,8 @@ export const viewTypes = {
   [ challengeTypes.step ]: 'step',
   [ challengeTypes.quiz ]: 'quiz',
   backend: 'backend',
-  modern: 'modern'
+  modern: 'modern',
+  invalid: 'invalid'
```

Comment:
This should be a top level route, there is no reason this should be in the Challenges route